### PR TITLE
{vis}[GCCcore/13.2.0] Wayland v1.22.0, PyCairo v1.25.1, ATK v2.38.0, ...

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)
- [x] requires #19152

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 10 out of 12 (12 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/29d176937c482ba3f83261b7c27117a2 for a full test report.

File: easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-8.2.2-GCCcore-13.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+homepage = 'https://www.freedesktop.org/wiki/Software/HarfBuzz'
+description = """HarfBuzz is an OpenType text shaping engine."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.2.0'}
+
+github_account = 'harfbuzz'
+source_urls = [GITHUB_SOURCE]
+sources = ['%(version)s.tar.gz']
+checksums = ['77c8c903f4539b050a6d3a5be79705c7ccf7b1cb66d68152a651486e261edbd2']
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/h/HarfBuzz/HarfBuzz-8.2.2-GCCcore-13.2.0.eb:14]:
```suggestion
checksums = ['0546aac7b2493b3681047914550860157f8799fc80bf5cb528927a9643d6ab3d']
```

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
@boegelbot please test @ jsc-zen2

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 18 out of 18 (13 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/b07b384351dc5841a0b101dfe0a9943e for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 13 out of 13 (13 easyconfigs in total)
alvis-c1 - Linux Rocky Linux 8.8, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/Micket/c94d431269202322236519db517947bc for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 13 out of 13 (13 easyconfigs in total)
zen2-ubuntu-2310 - Linux Ubuntu 23.10 (Mantic Minotaur), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.11.6
See https://gist.github.com/SebastianAchilles/5acd772b27818e74ed8c964ebcdf6bab for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @Micket!

**Pull request merged**